filter/store.go

```diff
@@ -41,16 +44,32 @@ type HTTPClient interface {
 }
 
 // PutCSVData allows the filtered file data to be sent back to the filter store when complete.
-func (store *Store) PutCSVData(filterJobID string, url string, size int64) error {
+func (store *Store) PutCSVData(filterJobID string, fileURL string, size int64) error {
 
-	// todo: make call to the filter API.
+	url := store.filterAPIURL + "/filters/" + filterJobID
 
```

Comment:
Request to filter API will need to send an `internal_token` header equal to the secret key, defaulted to `FD0108EA-825D-411C-9B1D-41EF7727F465`. Other than that the request url and body look fine to me.